src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,17 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStaus: (state) => {
+      state.categories = 'under construction';
+    },
+  },
+});
+
```

Comment:
**You did an awesome job!**

- [ ] One more step to get approved. So I can see you exported categoriesSlice.reduce. Also, we need to export `checkStatus` as an action. Like this: 

```javascript
export const { addBook, removeBook } = booksSlice.actions;
``` 
  Kindly export `checkStatus` as an action and be ready to get approved.

Reply:
Solved